test/TesterInternal/Serialization/BuiltInSerializerTests.cs

```diff
@@ -21,6 +21,8 @@
 
 namespace UnitTests.Serialization
 {
+    using TestGrainInterfaces;
+
     /// <summary>
```

Comment:
Instead of redefining `CircularTest1`, we use the implementation in `TestGrainInterfaces` because that implementation has a generated serializer whereas this assembly does not, due to `[assembly: SkipCodeGeneration]` attribute.